assembly/src/parsers/field_ops.rs

```diff
@@ -20,7 +20,7 @@ pub fn parse_add(op: &Token) -> Result<Node, ParsingError> {
         0 => unreachable!(),
         1 => Ok(Instruction(Add)),
         2 => {
-            let imm = parse_element_param(op, 1)?;
+            let imm = get_valid_felt(op, 1, parse_decimal_param(op, 1, op.parts()[1])?)?;
```

Comment:
nit: isn't it better for code readability to keep the `parse_decimal_param` in an immediate previous line? otherwise this can be too dense